api/app/Policies/CommunityPolicy.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Policies;
+
+use App\Models\User;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class CommunityPolicy
+{
+    use HandlesAuthorization;
+
+    /**
+     * Determine whether the user can view any models.
+     * Everyone is allowed to view the communities
+     *
+     * @return \Illuminate\Auth\Access\Response|bool
+     */
+    public function viewAny(?User $user)
+    {
+        if ($user) {
+            return $user->isAbleTo('view-any-community');
+        }
+
+        return true;
+    }
```

Comment:
Was a bit iffy on this, is it even worth checking the permission? Or should it just return true by default? 
But that makes it sort of pointless having those permissions to begin with 🤔

Reply:
@tristan-orourke can probably give a better answer but if anonymous (seems to be the case) users can view it, then I think just returning `true` makes sense. That is what we do for other things like `Skill` model.

https://github.com/GCTC-NTGC/gc-digital-talent/blob/b160f92cb9b11b95cb2af66ab41514aeed35c18f/api/app/Policies/SkillPolicy.php#L19-L22

Reply:
Yeah, I would say follow what the Skill and Department models use, which is just returning true.

Reply:
Okay, will this be fine @esizer 
https://github.com/GCTC-NTGC/gc-digital-talent/pull/10305/commits/f01e627ff21a48f4f05bc6cc90c1d3d5599f642e